stripper/ze_desert_fox_v7_6f.cfg

```diff
@@ -0,0 +1,21 @@
+;Making teleport to the boss arena humans only since boss is already hard enough to deal with.
+modify:
+{
+	match:
+	{
+		"targetname" "tel_ct_fin"
+		"classname" "trigger_teleport"
+	}
+	insert:
+	{
+		"filtername" "filter_ct"
+	}
+}
+
+add:
+{
+	"classname" "filter_activator_team"
+	"filterteam" "3"
+	"targetname" "filter_ct"
+	"Negated" "0"
+}
```

Comment:
This can be removed, will be merging in another change soon that more effectively simplifies the map.

Reply:
Done.